services/licenseStore.js

```diff
@@ -25,25 +25,68 @@ function buildPortedLicense(license, string, url) {
   return new License({ ...license, name, groups, url });
 }
 
+// Returns an index of licenses by license-id.
+//
+// Example:
+// {
+//   'cc-by-sa-3.0': { id: ..., name: ..., ... },
+// }
 function buildLicensesIndex(licenses) {
-  return licenses.reduce((idx, license) => Object.assign(idx, { [license.id]: license }), {});
+  return licenses.reduce(
+    (licensesIndex, license) => Object.assign(licensesIndex, { [license.id]: license }),
+    {}
+  );
+}
+
+// Returns an index of license-ids by license name.
+//
+// Example:
+// {
+//   'CC BY-SA 3.0': ['cc-by-sa-3.0', 'cc-by-sa-3.0-ported'],
+// }
+function buildLicenseNamesIndex(licenses) {
+  return licenses.reduce((licenseNamesIndex, license) => {
+    const licensesWithSameName = licenseNamesIndex[license.name] || [];
+    licensesWithSameName.push(license.id);
+    return Object.assign(licenseNamesIndex, { [license.name]: licensesWithSameName });
+  }, {});
```

Comment:
@tessi I updated the function variables' names with your suggestions and got rid of this weird if-else clause 😄 Also I kept the `Object.assign`-approach since it saves one line compared to this:
```js
licenseNamesIndex[license.name] = licensesWithSameName;
return licenseNamesIndex;
```